src/state/profile/hooks.ts

```diff
@@ -18,47 +17,32 @@ export const useFetchProfile = () => {
   }, [account, dispatch])
 }
 
-export const useProfileForAddress = (address: string) => {
```

Comment:
IN A MASSIVE PLOT TWIST I WOLUD LIKE TO KEEP THIS HOOK.

I want to fetch Profile information (avatar, username, achievements, points, team) for non-connected wallets.

This is just a nice wrapper around the `getProfile` function